publish/publish.proj

```diff
@@ -226,6 +233,48 @@
                                  IgnoredErrorMessagesWithConditional="@(IgnorableErrorMessages)" />
   </Target>
 
+  <Target Name="DoPushCoreHostPackagesToAzure"
+          DependsOnTargets="DoPushStandardPackagesToAzure;DoPushSymbolPackagesToAzure"
+          Condition="'$(OfficialPublish)' == 'true'"/>
+
+  <Target Name="DoPushStandardPackagesToAzure">
+    <PropertyGroup>
+      <RelativePath>$(PackagesRelativePath)</RelativePath>
+      <RelativePath Condition="'$(RelativePath)' == ''">$(ConfigurationGroup)/pkg</RelativePath>
+      <ExpectedFeedUrl>https://$(TransportFeedAccountName).blob.core.windows.net/$(TransportFeedContainerName)/$(RelativePath)/index.json</ExpectedFeedUrl>
```

Comment:
I'm surprised deriving the feed url is not part of the feed api

Reply:
I believe the api just expects a URL, @karajas is that correct?

Reply:
The URL needs to be specified currently.

Reply:
It shouldn't be hardcoded in the task library I actually expect this to be passed in from the build definition though not set here.